pkg/repositories/models/resource.go

```diff
@@ -18,11 +18,11 @@ type Resource struct {
 	CreatedAt    time.Time
 	UpdatedAt    time.Time
 	DeletedAt    *time.Time `sql:"index"`
-	Project      string     `gorm:"unique_index:resource_idx" valid:"length(0|255)"`
-	Domain       string     `gorm:"unique_index:resource_idx" valid:"length(0|255)"`
-	Workflow     string     `gorm:"unique_index:resource_idx" valid:"length(0|255)"`
-	LaunchPlan   string     `gorm:"unique_index:resource_idx" valid:"length(0|255)"`
-	ResourceType string     `gorm:"unique_index:resource_idx" valid:"length(0|255)"`
+	Project      string     `gorm:"uniqueIndex:resource_idx" valid:"length(0|255)"`
+	Domain       string     `gorm:"uniqueIndex:resource_idx" valid:"length(0|255)"`
+	Workflow     string     `gorm:"uniqueIndex:resource_idx" valid:"length(0|255)"`
+	LaunchPlan   string     `gorm:"uniqueIndex:resource_idx" valid:"length(0|255)"`
+	ResourceType string     `gorm:"uniqueIndex:resource_idx" valid:"length(0|255)"`
```

Comment:
Are these equivalent? or did we never have uniqe_index correctly? or is it deprecated...?

Reply:
@EngHabu unique_index worked in v1 but in v2 they changed it uniqueIndex

https://gorm.io/docs/indexes.html#uniqueIndex
https://v1.gorm.io/docs/models.html

This is one of the other breaking changes along with auto_increment which we faced during v2 migration

https://gorm.io/docs/v2_release_note.html#Breaking-Changes

Should have checked this out during the migration :)